contrib/fwupd.spec.in

```diff
@@ -165,11 +165,7 @@ Data files for installed tests.
 %else
     -Dplugin_dummy=false \
 %endif
-%if 0%{?enable_flashrom}
     -Dplugin_flashrom=true \
-%else
-    -Dplugin_flashrom=false \
-%endif
```

Comment:
I would think that you want to leave some of this in for guarding against rhel7 builds no?

Reply:
> against rhel7 builds

RHEL 7 and 8 have a very different .spec file already unfortunately.